aem-intellij-lang/src/main/kotlin/com/aemtools/lang/htl/file/HtlFileType.kt

```diff
@@ -28,16 +27,13 @@ object HtlFileType
   }
 
   override fun isMyFileType(file: VirtualFile): Boolean {
-    val project = ProjectLocator.getInstance().guessProjectForFile(file)
-
     if (file.isDirectory
-        || project == null
         || file.extension != "html") {
       return false
     }
 
     val path = file.path
-    return HtlDetectionService.isHtlFile(path, project)
```

Comment:
this change broke, Mark Directory as HTL Root

Reply:
@bobi Thank you for your comment. I will investigate and fix this bug.

Reply:
@bobi , I would be grateful if you could provide steps on how to reproduce this issue.

Reply:
Here is original feature request https://github.com/aemtools/aemtools/issues/115.
Steps to reproduce:
1. put html files in any folder except "jcr_root", e.g. "resources"
2 Mark Directory as HTL Root.

Expected behavior: html files under "resources" directory should be editable by plugin, but now they are editable by default html editor.

https://monosnap.com/file/ZdybOMB5fpBj5j0MogiwFXbn5tWRHE

Reply:
i have opened issue https://github.com/aemtools/aemtools/issues/214

Reply:
Thank you for your findings. I will prioritize it later. You will receive a notification about a status update.

Reply:
i don't know your case, but temporary i have fixed for myself in following way
```
override` fun isMyFileType(file: VirtualFile): Boolean {
    if (file.isDirectory
        || file.extension != "html") {
      return false
    }

    val project = ProjectLocator.getInstance().guessProjectForFile(file)
    val path = file.path

    return if (project == null) {
      HtlDetectionService.isHtlFile(path)
    } else {
      HtlDetectionService.isHtlFile(path, project)
    }
  }
```

Reply:
Fixed in https://github.com/aemtools/aemtools/pull/217.